LayoutTests/ipc/send-gpu-GetShareableBitmap-RemoteRenderingBackend-expected.txt

```diff
@@ -0,0 +1,5 @@
+layer at (0,0) size 800x600
```

Comment:
The test should be marked as text test, and you should print out something like "DONE" so that it's clear that the test ran and did not crash. You can take a look at the other similar tests